### PR TITLE
JA: Translated site/ja/beta/tutorials/eager/custom_layers.ipynb

### PR DESCRIPTION
**Pull request by masa-ita**

### CODE REVIEW

File: site/ja/beta/tutorials/eager/custom_layers.ipynb [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+   },
+   "outputs": [],
+   "source": [
+    "# tf.keras.layers パッケージの中では、レイヤーはオブジェクトです。\n",
+    "# レイヤーを構築するためにすることは、単にオブジェクトを作成するだけです。\n",
+    "# ほとんどのレイヤーでは、最初の引数が出力の次元あるいはチャネル数を表します。\n",
+    "layer = tf.keras.layers.Dense(100)\n",
+    "# 入力の次元数は多くの場合不要となっています。それは、レイヤーが最初に使われる際に\n",
+    "# 推定可能だからです。ただし、複雑なモデルを構築する場合などでは、\n",
+    "# 手動で設定するために、引数として渡してやることも可能です。\n",
```
Comment (AseiSugiyama) [inline site/ja/beta/tutorials/eager/custom_layers.ipynb]: [suggestion]

```
# 推定可能だからです。ただし、複雑なモデルを構築する場合などでは、
# 手動で設定するために、引数として渡してやることも可能です。
==>
# 推定可能だからです。ただし、引数として渡すことで手動で指定することも可能です。
# これは複雑なモデルを構築する場合に役に立つでしょう。
```

File: site/ja/beta/tutorials/eager/custom_layers.ipynb [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+   "metadata": {
+    "colab_type": "text",
+    "id": "O0kDbE54-5VS"
+   },
+   "source": [
+    "## カスタムレイヤーの実装\n",
+    "\n",
+    "独自のレイヤーを実装する最良の方法は、tf.keras.Layer クラスを拡張し、下記のメソッドを実装することです。\n",
+    "  *  `__init__` , 入力に依存しないすべての初期化を行う\n",
+    "  * `build`, 入力の形状を知った上で、残りの初期化を行う\n",
```
Comment (AseiSugiyama) [inline site/ja/beta/tutorials/eager/custom_layers.ipynb]: [suggestion]
The word `shapes` used here is a technical term of Keras and Japanese translation of Keras Documentation [does not translate the word `shape(s)`](https://keras.io/ja/getting-started/sequential-model-guide/#shape).

I guess the below is better for consistency:
`入力のshapeを知った上で、残りの初期化を行う`

File: site/ja/beta/tutorials/eager/custom_layers.ipynb [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+   "metadata": {
+    "colab_type": "text",
+    "id": "Qhg4KlbKrs3G"
+   },
+   "source": [
+    "## モデル：レイヤーの組み合わせ\n",
+    "\n",
+    "機械学習では、多くのレイヤーに類するものが、既存のレイヤーを組み合わせることで実装されています。例えば、ResNetの残差ブロックは、畳込み、バッチ正規化とショートカットの組み合わせです。\n",
+    "\n",
+    "他のレイヤーからなるレイヤーに類するものを定義する際の主役は、tf.keras.Model クラスです。このクラスを継承することで実装することができます。"
```
Comment (AseiSugiyama) [inline site/ja/beta/tutorials/eager/custom_layers.ipynb]: I guess `できます` is better than `することができます`.

File: site/ja/beta/tutorials/eager/custom_layers.ipynb [hunk 4]
```diff
@@ -0,0 +1,402 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "wYfucVw65PMj"
+   },
+   "source": [
+    "しかし、ほとんどの場合には、モデルはレイヤーを次々に呼び出すことで構成されます。tf.keras.Sequential クラスを使うことで、これを最少の行数で行うことができます。"
```
Comment (AseiSugiyama) [inline site/ja/beta/tutorials/eager/custom_layers.ipynb]: [suggestion]
```
これを最少の行数で行うことができます
==>
これをかなり短いコードで実装できます。
```

File: site/ja/beta/tutorials/eager/custom_layers.ipynb (hunk 1 above)
Comment (masa-ita) [reply site/ja/beta/tutorials/eager/custom_layers.ipynb]: Thank you.  It's better!  I fixed it.

File: site/ja/beta/tutorials/eager/custom_layers.ipynb (hunk 2 above)
Comment (masa-ita) [reply site/ja/beta/tutorials/eager/custom_layers.ipynb]: Thanks.  I've modified it.

File: site/ja/beta/tutorials/eager/custom_layers.ipynb (hunk 3 above)
Comment (masa-ita) [reply site/ja/beta/tutorials/eager/custom_layers.ipynb]: Thank you.  I agree.

File: site/ja/beta/tutorials/eager/custom_layers.ipynb (hunk 4 above)
Comment (masa-ita) [reply site/ja/beta/tutorials/eager/custom_layers.ipynb]: Thank you.  It's better.  I fixed it.

**Comment by masa-ita** [comment]
I've rebased and move custom_layer.ipynb from site/ja/beta/tutorials/eager/ to site/ja/tutorials/customization/.

**Comment by masa-ita** [comment]
@lamberta Thank you.  I'm sorry to forget to fix them.

**Pull request merged**